sport/app/football/model/CompetitionStages.scala

```diff
@@ -214,12 +214,12 @@ object KnockoutSpider {
       //   3 ━┓   ┃   ┃   ┏━ 7
       //      ┣━━━┛   ┗━━━┫
       //   4 ━┛           ┗━ 8
-      ZonedDateTime.of(2024, 6, 29, 17, 0, 0, 0, ZoneId.of("Europe/London")), // Match 38
-      ZonedDateTime.of(2024, 6, 30, 17, 0, 0, 0, ZoneId.of("Europe/London")), // Match 40
-      ZonedDateTime.of(2024, 7, 1, 20, 0, 0, 0, ZoneId.of("Europe/London")), // Match 41
-      ZonedDateTime.of(2024, 7, 1, 17, 0, 0, 0, ZoneId.of("Europe/London")), // Match 42
       ZonedDateTime.of(2024, 6, 30, 20, 0, 0, 0, ZoneId.of("Europe/London")), // Match 39
```

Comment:
Switching line 217 and 218 would bring this in line to the print version

Reply:
Cannot be done! Game 39 is the "home" for the quarter-final, so the order has to stay as it is

Reply:
While that is correct, as we do not have control of the order of the resulting match – 39 is on top, 37 is on bottom – it would probably create more confusion:

<img width="285" alt="image" src="https://github.com/guardian/frontend/assets/76776/900c091e-ddd0-4c35-a279-7a808166ca2d">

In that spirit I think it’s best to keep as-is